pkg/crc/config/settings.go

```diff
@@ -70,7 +70,7 @@ func RegisterSettings(cfg *Config) {
 
 	// Preset setting should be on top because CPUs/Memory config depend on it.
 	cfg.AddSetting(Preset, string(preset.OpenShift), validatePreset, RequiresDeleteAndSetupMsg,
-		fmt.Sprintf("Virtual machine preset (valid values are: %s, %s and %s)", preset.Podman, preset.OpenShift, preset.OKD))
+		fmt.Sprintf("Virtual machine preset (valid values are: %s)", preset.AllPresets()))
```

Comment:
What does the string looks like ? I don't know how go converts an array to a string.

Reply:
@cfergeau it have `Virtual machine preset (valid values are: [openshift okd podman microshift])` if we really want to have as string then another helper is required.

Reply:
`strings.Join(preset.AllPresets, ", ")` might work.

Reply:
> strings.Join(preset.AllPresets, ", ") might work